src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/CountA.txt

```diff
@@ -0,0 +1,338 @@
+// ******** TABLE WITH NO BLANK RECORDS ********
+
+>> CountA([0])
+1
+
+>> CountA([1])
+1
+
+// NUMERIC RECORDS
+
+>> CountA([1, 2, 3, 4, 5])
+5
+
+>> CountA(Table({a:1},{a:2},{a:3}))
+3
+
+// STRING RECORDS
+
+>> CountA(["1", "one", "two", "three", "four"])
+5
+
+>> CountA(Table({a:"1"},{a:"two"},{a:"three"},{a:"four"},{a:"five"}))
+5
+
+// BOOLEAN RECORDS
+
+>> CountA([true, false])
+2
+
+>> CountA(Table({a:true},{a:false}))
+2
+
+// DATE/TIME/DATETIMEVALUE RECORDS
+
+>> CountA([Date(2022,12,12), DateValue("10/01/2014")])
+2
+
+>> CountA(Table({a:Date(2022,12,12)},{a:DateValue("10/01/2014")}))
+2
+
+>> CountA([DateTimeValue("5/12/2022 6:30:30 PM"), DateTimeValue("May 19, 2022")])
+2
+
+>> CountA(Table({a:DateTimeValue("5/12/2022 6:30:30 PM")},{a:DateTimeValue("May 19, 2022")}))
+2
+
+//  ALL DATATPYE RECORDS 
+
+﻿>> CountA([1, "2", "two", true, false, Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM")])
+#Error(Kind=InvalidArgument)
+
+﻿>> CountA([1, "2", true, false, Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM")])
+7
+
+// ******** TABLE WITH NO BLANK RECORDS ********
+
+﻿>> CountA([0,""])
+1
+
+>> CountA(["",""," ",1]
+4
+
+>> CountA([1,2,3,4,Blank()])
+4
+
+﻿>> CountA([1, "2", true, false, Blank(), "", " ", Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM")])
+#Error(Kind=InvalidArgument)
+
+>> CountA([1,"2", true, Blank(),Blank(),Blank()])
+3
+
+>> CountA([1,Date(2022,12,12),"3",If(1<0,4),Blank()])
+3
+
+>> CountA(Table({a:false},{a:Blank()},{a:true},{a:Blank()}))
+2
+
+>> CountA(Table({a:"1"},{a:"2"},If(1<0,{a:"3"}),If(1>0,{a:"4"}),{a:"5"}))
+4
+
+>> CountA(LastN([1,2,3,4], Blank()))
+Blank()
+
+>> CountA([LastN([1,2,3,4]), Blank()])
+1
+
+//One blank record
+>> CountA([Blank()])
+0
+
+//All blank records
+﻿>> CountA([Blank(),Blank(),Blank(),Blank(),Blank()])
+0
+
+>> CountA(Table({a:Blank()},{a:Blank()},{a:Blank()}))
+0
+
+﻿>> CountA(["",""," ",Blank(),Blank()])
+3

```

Comment:
If we want to treat blanks and empty strings the same, this should return 1 instead of 3. Same for next test

Reply:
But Excel also counts empty strings, so we need to decide what to do here

Reply:
Yes Excel also has the same behavior and returns 3.
@CarlosFigueiraMSFT Do you want me to file a bug for this or add this in the ExcelCompatibility file for discussion?

Reply:
Can you please add it to the excel compat file? Thanks!

Reply:
Added the scenario in Excel compat file